examples/train_simple_navigation_tf2.py

```diff
@@ -466,107 +484,88 @@ def _make_sample(*exps):
-            nn.LeakyReLU(),
-            nn.MaxPool2d(kernel_size=(2, 2), stride=(2, 2)),  # 21*21
-        )
-        calc_size = lambda x: (x // 2) // 2
-        latent_size = num_filters[1] * calc_size(input_shape[1]) * calc_size(
-            input_shape[2])
-        self.q_nn = nn.Sequential(
-            nn.Linear(latent_size, fc_size[0]),
-            nn.LeakyReLU(),
-            nn.Linear(fc_size[0], fc_size[1]),
-            nn.LeakyReLU(),
-            nn.Linear(fc_size[1], num_actions),
-        )
-        self.q_nn[-1].weight.data.fill_(0.0)
-        self.q_nn[-1].bias.data.fill_(0.0)
-        self.v_nn = nn.Sequential(
-            nn.Linear(latent_size, fc_size[0]),
-            nn.LeakyReLU(),
-            nn.Linear(fc_size[0], fc_size[1]),
-            nn.LeakyReLU(),
-            nn.Linear(fc_size[1], 1),
-        )
-        self.v_nn[-1].weight.data.fill_(0.0)
-        self.v_nn[-1].bias.data.fill_(0.0)
+                padding='same',
+                strides=1,
+                data_format='channels_last'),
+            tf.keras.layers.LeakyReLU(alpha=0.01),
+            tf.keras.layers.MaxPooling2D(pool_size=(2, 2), strides=(2, 2), data_format='channels_last'),
+            tf.keras.layers.Flatten(data_format='channels_last')
+            ], name = name + "/latent_nn")
+
+        self.latent_nn.summary()
+
+        self.q_nn = tf.keras.Sequential([
+            tf.keras.layers.Dense(fc_size[0], input_shape=self.latent_nn.output_shape[1:]),
+            tf.keras.layers.LeakyReLU(alpha=0.01),
+            tf.keras.layers.Dense(fc_size[1]),
+            tf.keras.layers.LeakyReLU(alpha=0.01),
+            tf.keras.layers.Dense(num_actions)
+            ], name = name + "/q_nn")
+        self.q_nn.summary()
+
+        self.v_nn = tf.keras.Sequential([
+            tf.keras.layers.Dense(fc_size[0], input_shape=self.latent_nn.output_shape[1:]),
+            tf.keras.layers.LeakyReLU(alpha=0.01),
+            tf.keras.layers.Dense(fc_size[1]),
+            tf.keras.layers.LeakyReLU(alpha=0.01),
+            tf.keras.layers.Dense(1)
+            ], name = name + "/v_nn")
+        self.v_nn.summary()
+
 
     def calc_q_values(self, state):
         latent = self.latent_nn(state)
-        latent = latent.reshape(latent.shape[0], -1)
         q_values = self.q_nn(latent)
 
         # Dueling Network: https://arxiv.org/pdf/1511.06581.pdf
         value = self.v_nn(latent)
-        mean_q = torch.mean(q_values, dim=-1, keepdim=True)
+        mean_q = tf.reduce_mean(q_values, axis=-1, keepdims=True)
         adjust = value - mean_q
         q_values = q_values + adjust
         return q_values
 
-    def __call__(self, state):
+    def call(self, state):
         return self.calc_q_values()
 
+    def show_parameter_stats(self):
+        pass
 
 if __name__ == "__main__":
     options = Options()
     os.makedirs(options.model_dir, exist_ok=True)
     logging.basicConfig(level=logging.INFO)
+    logging.getLogger().setLevel(logging.INFO)
```

Comment:
Adding a comment saying that the logger may have already been created by tensorflow and thus logging.basicConfig() in the previous line won't create a new logger hence `level=logging.INFO` in the previous line does not have any effect. So we need to explicitly set log level here to enable the logging.